packages/cli/src/lib/fast-glob.js

```diff
@@ -0,0 +1,10 @@
+const fg = require('fast-glob')
+
+async function listFilesMatchPattern(pattern) {
+  const files = await fg([pattern])
```

Comment:
As `fast-glob` accepts an array of glob patterns, it make sense to support multiple patterns in cli as well. That will enable us to do `bundle-validator check "dist/*.js" "es/*.js"`.